client/js/app.js

```diff
@@ -0,0 +1,57 @@
+const form = document.querySelector('#story-form');
+const storyList = document.querySelector('table')
+
+form.addEventListener('submit', submitStory);
+
+getAllStories();
+
+function getAllStories() {
+    fetch('http://localhost:3000/stories')
+        .then(r => r.json())
+        .then(appendStory)
+        .catch(console.warn)
+};
+
+function submitStory(e) {
+    e.preventDefault();
+
+    const storyData = {
+        title: e.target.title.value,
+        pseudonym: e.target.pseudonym.value,
+        body: e.target.body.value,
+    };
+
+    const options = {
+        method: 'POST',
+        body: JSON.stringify(storyData),
+        headers: { "Content-Type": "application/json" }
+    };
+
+    fetch('http://localhost:3000/stories', options)
+        .then(r => r.json())
+        .then(appendStory)
+        .then(() => e.target.reset())
+        .catch(console.warn)
+};
+
+function appendStory(storyData) {
+    const newRow = document.createElement('tr');
+    const storyLi = formatStoryTable(storyData, newRow)
```

Comment:
I believe this was a typo, considering that the next line is appending to 'storyList'